libs/damap/src/assets/i18n/dashboard/de.json

```diff
@@ -1 +1,74 @@
-{}
+{
+  "dashboard": {
+    "welcome.message": "Willkommen beim DMP Tool, einem Dienst, der Ihnen hilft, den Datenmanagementplan (DMP) für Ihr Projekt zu erstellen.",
+    "dmp": {
+      "heading": "Was ist ein Datenmanagementplan (DMP)?",
+      "text": {
+        "prefix": "Ein Datenmanagementplan (DMP) übersetzt datenbezogene Aspekte von ",
+        "dmp": {
+          "text": "DMP",
+          "link": "https://www.tuwien.at/en/research/rti-support/research-data/rdm-infos-tips/dmp"
```

Comment:
Does MUG provide their own link?

Reply:
@rekt-hard please make sure to check within the provided file.